### ISSUE TITLE
Google change terms: Switch to apps that use secure OAuth access - June 15, 2020

### ISSUE THREAD

**Issue by luizsculz** [RainLoop/rainloop-webmail#1957]
Rainloop version: 1.12.1, SO: Windows 10, Google Chrome: 79.0.3945.88 (x64), 
Google recently announced a change in its terms which will disable less secure application access, will this impact on using rainloop for gmail accounts?

Access to LSAs will be turned off in two stages:

June 15, 2020 - Users who try to connect to an LSA for the first time will no longer be able to do so. This includes third-party apps that allow password-only access to Google calendars, contacts, and email via protocols such as CalDAV, CardDAV and IMAP. Users who have connected to LSAs prior to this date will be able to continue using them until usage of all LSAs is turned off.
February 15, 2021 - Access to LSAs will be turned off for all G Suite accounts.

*** attached google email about it

![3](https://user-images.githubusercontent.com/2709453/71280200-55ee2580-2339-11ea-82a0-fb7153ecd52d.png)
![4](https://user-images.githubusercontent.com/2709453/71280201-55ee2580-2339-11ea-9703-1a3c6044388b.png)
![1](https://user-images.githubusercontent.com/2709453/71280202-5686bc00-2339-11ea-8c7e-e91729ab5cc5.png)
![2](https://user-images.githubusercontent.com/2709453/71280203-5686bc00-2339-11ea-8440-2c09180e557d.png)

**Comment by Faillinx** [comment]
Yes this will impact rainloops function. We have updated to 1.14 and are trying to get it to work with gsuite. I can login once via oauth, however it does not function a second time. Not sure how to troubleshoot or fix this.

**Comment by ScrumpyJack** [comment]
OAuth support would be fantastic. Does anyone know if it's being developed?
I respect @RainLoop 's choice of not engaging the rainloop community of users, not getting involved in the issues log here on github, any pushing commits only at release time, but it does make it quite difficult to figure out a roadmap.

**Comment by RainLoop** [comment]
> OAuth support would be fantastic. 

We already support this.

![image](https://user-images.githubusercontent.com/5530092/105030728-4625a100-5a65-11eb-8f8b-d6e4ae1a5d3d.png)

**Comment by Faillinx** [comment]
We had issues getting it to work a year or so ago when we last tried it. It would login the first time, and then not ever login again until the account was reset in the gsuite panel. Sorry I don't know more details other than it forced us to look for another solution.

**Comment by ScrumpyJack** [comment]
@RainLoop Nice to hear from you!
Aha! I had not seen this.
Any chance you could move that to a generic OAUTH2 so that we can use it with Microsoft, or Yahoo! or any other mail service provider that requires OAUTH2 to connect?

**Comment by luizsculz** [comment]
Hello Microsoft will close the traditional login (with password) on 12/31/22, from that date Rainloop will no longer work for Microsoft users, if you don't have Oauth2, any plans to implement this?
